docs/faq.rst

```diff
@@ -239,7 +234,7 @@ higher) in your spider:
 
 
     class MySpider(CrawlSpider):
-        name = "myspider"
+        name = "my_spider"
```

Comment:
Run a text replacement to make the naming consistent. I am OK with going the other way: `my_spider` → `myspider`.